install/docker-compose.yaml

```diff
@@ -27,9 +27,11 @@ services:
       - traefik.enable=true
       - traefik.http.routers.php-fpm.tls=true
       - traefik.http.routers.php-fpm.rule=Method(`GET`,`POST`,`HEAD`,`PUT`,`PATCH`,`DELETE`)  # Poor man's default route
+    # volumes:
+    #   - I:\Extra\wikijump:/var/www
```

Comment:
Commented code.

Reply:
Until we really firm up local dev environments I'll settle for changing this into an informational thing. Mapping a local volume for the web container is super duper useful.